CameraOpenCV/OcvOp.cs

```diff
@@ -25,13 +26,10 @@ public void Blur(SoftwareBitmap input, SoftwareBitmap output, Algorithm algorith
                 using Mat mInput = SoftwareBitmap2Mat(input);
                 using Mat mOutput = new Mat(mInput.Rows, mInput.Cols, MatType.CV_8UC4);
 
-                //App.CvHelper.T
-
                 Cv2.Blur(mInput, mOutput,
-                    (Size)algorithm.AlgorithmProperties[0].CurrentValue,
-                    (Point)algorithm.AlgorithmProperties[1].CurrentValue,
-                    (BorderTypes)algorithm.AlgorithmProperties[2].CurrentValue);
-                //Cv2.ImShow("Blur", mOutput);
+                    (Size)algorithm.AlgorithmProperties[0].CurrentValue,            // ksize
```

Comment:
I think it's more difficult to make mistakes to use named parameters instead of comments.
```suggestion
                    ksize: (Size)algorithm.AlgorithmProperties[0].CurrentValue, 
```